nebullvm/operations/optimizations/compilers/quantizations/utils.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from nebullvm.tools.base import QuantizationType
+
+logger = logging.getLogger("nebullvm_logger")
```

Comment:
sometimes we use "nebullvm_logger", others just "nebullvm". Please be consistent